tests/test_cpmorphology.py

```diff
@@ -3031,9 +3031,9 @@ def test_01_04_random(self):
             "GlfbnTMQ6D/G0/uKnEjrP7RqrXnOdMM/4A8ROHauxT+W/BqsnbDSP4xRkmj"
             "HxNU/TugfQWuE5z8="
         )
-        data = np.fromstring(data)
+        data = np.frombuffer(data)
         data.shape = (40, 40)
-        expected = np.fromstring(expected)
+        expected = np.frombuffer(expected)
```

Comment:
Can you confirm that this change works in `python 3.9` with `numpy < 2` AND `python 3.13` with `numpy >= 2`